### PR TITLE
chore: upgrade omniauth from '~> 1.5' to '~> 2.0'

### PR DESCRIPTION
**Pull request by EtienneM**
Related to the CVE-2015-9284. The fix is in omniauth 2.0.0: https://github.com/omniauth/omniauth/releases/tag/v2.0.0.

I tested this change in dev and it works perfectly :) 

@ksol I may need help to make the GitHub Action publish work

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```
Comment (Soulou) [inline .github/dependabot.yml:3]: Isn't it `bundle`?

Comment (EtienneM) [reply .github/dependabot.yml:3]: As this PR enables GitHub Action on this project, we want to enable Dependabot for the GitHub Action dependencies.

And as this project is a gem we don't have a `Gemfile.lock`. Constraints seems to be declared in a `gemspec` file (https://github.com/Scalingo/omniauth-scalingo/blob/master/omniauth-scalingo.gemspec#L19-L24) and I'm not sure if it makes it possible to enable Dependabot also for the Ruby dependencies.

**Review by ksol — COMMENTED** [review]
we might want to do a major bump of the gem? it's significant enough

**Comment by EtienneM** [comment]
> we might want to do a major bump of the gem? it's significant enough

Yep OK :)

**Review by ksol — APPROVED** [review]

**Pull request merged**